course_catalog/etl/video_test.py

```diff
@@ -0,0 +1,126 @@
+"""Tests for Video ETL functions"""
+# pylint: disable=redefined-outer-name
+import pytest
+from django.conf import settings
+from course_catalog.etl import video
+
+
+settings.VIDEOS_CONFIG_FILE_LOCATION = "https://youtube.yaml"
```

Comment:
It's best to move this into a fixture because this will get executed when the module loads which might have some negative side effects. `mocked_channel_data` is a fine place to move it.